tests/discrete_gauss.rs

```diff
@@ -20,12 +20,13 @@ use serde::Deserialize;
 #[derive(Debug, Eq, PartialEq, Deserialize)]
 pub struct DiscreteGaussTestVector {
     #[serde(with = "hex")]
-    seed: [u8; 16],
+    seed: [u8; 32],
     std_num: u128,
     std_denom: u128,
     samples: Vec<i128>,
 }
 
+#[ignore = "need to upgrade the test vector code to VDAF-13"]
```

Comment:
@divergentdave do you wanna upgrade your gist and re-generate test vectors real quick?

Reply:
Updated, the test passes with fresh test vectors

Reply:
Done.